nevergrad/optimization/optimizerlib.py

```diff
@@ -611,19 +611,16 @@ def _internal_tell_not_asked(self, candidate: p.Parameter, loss: tp.FloatLoss) -
         raise base.TellNotAskedNotSupportedError
 
 
-@registry.register
 class PCEDA(EDA):
```

Comment:
Why not delete them directly if no more needed?

Reply:
I'd like to be sure that nobody will miss this... @fteytaud ?